text/0092-replay-issue-creation.md

```diff
@@ -0,0 +1,55 @@
+- Start Date: 2023-05-15
+- RFC Type: decision
+- RFC PR: https://github.com/getsentry/rfcs/pull/92
+- RFC Status: draft
+
+# Summary
+
+We want to detect certain categories of issues only available through the Session Replay product. These issues can only be detected on the SDK. The Replay back-end will never have enough data to find these issues. For that reason this is primarily an SDK driven workload. The question is: what role should the Replay back-end have in Replay issue creation? Should the Replay SDK use the Replay back-end to generate new issues or should the SDK generate those issues through a generic, non-replay-specific interface?
+
+# Motivation
+
+1. Inform the long-term strategy for detecting and ingesting Session Replay generated issues.
+2. Provide actionable feedback to developers that could not otherwise be captured in a performance span or error event.
+3. Increase product awareness in free-tier customers and encourage adoption.
+
+# Options Considered
+
+### Option 1: SDK
+
+When the SDK encounters a "replay issue" it will make an HTTP request to a generic interface which will handle the issue creation process.
+
+**Pros:**
```

Comment:
For each option, I would split out the Pros/Cons from engineering POV vs. product POV. (I can fill in Product ✨ ) 

A clear "Pro" to this solution is we'd be exposing Dead Click Issues to much much larger customer segment when compared to only Replay customers, for example. 😃

Reply:
Added.  I left it as a single pros/cons table for now.  If we start to accumulate a lot of items we can split it.